src/amo/reducers/reviews.js

```diff
@@ -91,6 +94,8 @@ export type ReviewsState = {|
   view: {
     [reviewId: number]: ViewStateByReviewId,
   },
+  // Short-lived messages about reviews.
+  flashMessage?: FlashMessageType,
```

Comment:
I see this could be undefined but even so should be added to the initial Review State? or is that intentionally left out b/c of this?

Reply:
Yeah, probably a good idea. Done.